src/test/java/seedu/jxmusic/logic/commands/AddCommandTest.java

```diff
@@ -122,7 +122,7 @@ public void updatePerson(Person target, Person editedPerson) {
         }
 
         @Override
-        public void updateFilteredPersonList(Predicate<Person> predicate) {
+        public void updateFilteredTrackList(Predicate<Track> predicate) {
```

Comment:
Not sure if this should be: `public void updateFilteredPlaylistList(Predicate<Playlist> predicate) {`